hack/functest.sh

```diff
@@ -1,3 +1,8 @@
-#!/bin/bash -e
+#!/usr/bin/env bash
 
-echo hello world
+set -xe
+
+source ./cluster/cluster.sh
+
+export KUBEVIRT_CLIENT_GO_SCHEME_REGISTRATION_VERSION=v1
+KUBECONFIG=${KUBECONFIG:-$(cluster::kubeconfig)} $GO test -test.timeout=1h -race -test.v ./tests/... $E2E_TEST_ARGS -ginkgo.v
```

Comment:
we have twice `-v`, lets remove the deprecated one

Reply:
which one of the is deprecated?

Reply:
I think the ginkgo one
best to try please the different ways